README.md

```diff
@@ -71,3 +71,11 @@ class with child classes for the various permutations of TRACK3R
 * TRACK3RWithBallShooter
 * TRACK3RWithClaw
 * TRACK3RWithSpinner
+
+## More robot programs
+
+The [LEGO Mindstorms EV3 Comparison
+project](https://github.com/bittner/lego-mindstorms-ev3-comparison) sports a
+few other robot programs in addition. Its [comparison
+table](https://github.com/bittner/lego-mindstorms-ev3-comparison#inspiration-for-lego-ev3-robots)
```

Comment:
Note that if you prefer shorter lines of code you can also cut off links in MarkDown after the opening brace, e.g.
```
Its [comparison table](
https://...very...very...very...loooooooooooog...url/...)
also lists ...
```